terraform/main.tf

```diff
@@ -0,0 +1,22 @@
+provider "apigee" {
+  org          = var.apigee_organization
+  access_token = var.apigee_token
+}
+
+terraform {
+  backend "azurerm" {}
+
+  required_providers {
+    apigee = "~> 0.0"
+    archive = "~> 1.3"
+  }
+}
+
+module "hello-world" {
+  source             = "github.com/NHSDigital/api-platform-service-module?ref=apm-501-change-proxy-location"
```

Comment:
Can remove the ref on this as the branch has since been merged

Reply:
changed